content/install_bootstrap.md

```diff
@@ -18,6 +18,48 @@ aliases = ["/install_bootstrap.html"]
 
 {{% chef_client_bootstrap_stages %}}
 
+## Minimum Recommended Recipe Run-List on a Node
+
+You do not have to choose a complete recipe [run-list](/run_lists/) now if you are not sure which
+recipes you want to bootstrap a node with. When running Chef Infra Client 15.x,
+we recommend as your minimum run-list the cron resource in the
+[chef-client cookbook](https://supermarket.chef.io/cookbooks/chef-client)
+which will help you control how and when the `chef-client` command will run and
```

Comment:
```suggestion
recipes you want to bootstrap a node with. When running Chef Infra Client 15.x, at a minimum
we recommend adding the cron resource in the
[chef-client cookbook](https://supermarket.chef.io/cookbooks/chef-client) to your run-list.
This will help you control how and when the `chef-client` command will run and
```